bears/general/LineContinuationBear.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.bearlib.languages.Language import Language
+
+
+class LineContinuationBear(LocalBear):
+    def run(self,
+            filename,
+            file,
+            language,
+            ):
+        try:
+            lang = Language[language]
+            line_continuation = lang.get_default_version().line_continuation
+        except Exception:
+            logging.error('Language {} is not yet supported.'.format(language))
+            return
+
+        for line_number, line in enumerate(file):
+            if len(line) - 1:
+                if line[-2] in line_continuation:
```

Comment:
If there is trailing whitespace after line_continuation delimiter, your bear cannot detect

Reply:
Yeah, but I guess that is already handled by another bear. So I guess these two conditions will never overlap, even if they do, one of the bears will throw an error.

Besides, as stated here https://github.com/coala/coala-bears/issues/2427#issuecomment-434286503 the initial idea of this bear is same.

Reply:
Well, IMO it is invalid to have other characters following line continuation delimiter, so it might be fine.